### DISCUSSION TITLE
Potential paper theme

### DISCUSSION THREAD

**Discussion by kjappelbaum** [MADICES/MADICES-2022#34] [Ideas]
I'm not in for writing a paper just for the sake of writing a paper and it would be good to have a theme to focus the breakouts a bit better. 

Potential themes: 
- Key challenges and possibilities of machine actionable data for materials science? 
- What is needed to make XAS/XPS data machine-actionable?

**Comment by ml-evs** [comment]
I think my questions are:

1. What would be the best format? We could write a short perspective (<5 pages, feasible with just the organizers), more of a review of existing work to motivate future work (10+ pages, would need at least some input from others) and/or a living document (that could live in this repo, would rely on lots of input from others) more in line with projects like The Turing Way.
2. What perspective do we bring that is different from what is out there? I would say the focus on data capture and schema discoverability as requirements for both of your potential themes above.

Regarding XAS specifically, if we can get the XAS experts onboard then sure, but not sure we are qualified to write an XAS-specific paper with just the organizers, unless we are again just writing something general and using XAS for specific examples.

I know @shyamd had thoughts on this too.

**Reply by rlaplaza** [reply]
I think that a short perspective calling for MADICES altogether  would be a good read for the broader community, whereas a XAS-specific paper seems a bit out of scope. I agree with point 2 above, but in a more general sense I think that the "push" for 4 and 5 stars in chemistry and materials science (in the spirit of Tim Berners-Lee) is not out there altogether, which makes for a good driving theme.

**Reply by eibfl-dtu** [reply]
Agree, MADICES showed the variety of tools out there to handle all aspects of making data FAIR: describe it (metadata and ELNs), structure it (schemas and formats), share it (API specifications) and link it (ontologies). I think it could be valuable a small perspective just listing these main players, and offering perspectives on common challenges, how to integrate them and exploit them. I would be interested in contributing :)

**Reply by kjappelbaum** [reply]
>whereas a XAS-specific paper seems a bit out of scope

yea, we didn't end up talking about x-ray a lot :D

**Comment by kjappelbaum** [comment]
The review like elements we had (the awesome list, glossary, ...) could potentially be fleshed out into a "living review" in something like 
https://github.com/livecomsjournal
